src/app/Components/LanguageAvatar.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  export let name: string;
+  export let selectedLanguage: string = "";
+
+  export let isSelected: boolean = name === selectedLanguage;
+  export let className: string = "avatar";
+  if (isSelected) {
+    className = className + " selected";
+  }
```

Comment:
```suggestion
  export let className: string = "avatar";
```

Additionally, is the `className` supposed to be modifiable from the outside? If not, then I would remove `export` as well:

```suggestion
  let className: string = "avatar";
```